BUILD.bazel

```diff
@@ -194,6 +194,18 @@ cc_binary(
     deps = ["//src/google/protobuf/compiler:protoc_lib"],
 )
 
+proto_toolchain(
```

Comment:
`proto_toolchain` is a macro that already creates a `toolchain` target for you.
https://github.com/bazelbuild/rules_proto/blob/main/proto/private/rules/proto_toolchain.bzl#L34